app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
 end
```

Comment:
- [ ] You currently do not have user's controller. Please Implement the user's controller with the following actions:

`show action`

- [ ] Lists events created by user, i.e. `@created_events = current_user.events`
- [ ] Lists events previously attended by user using event model scope, i.e. `@past_events = current_user.attended_events.previous`
- [ ] Lists upcoming events previously attended by user using event model scope, i.e. `@upcoming_events = current_user.attended_events.upcoming`